### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by rhrazdil**
Bump dependencies.

**What this PR does / why we need it**:

**Special notes for your reviewer**:

When trying to use `k8s.io/kubelet v0.19.1`, `make check` fails on
```
pkg/deviceplugin/lister.go:168:31: cannot use NewMacvtapDevicePlugin(c.Name, c.Master, c.Mode, c.Capacity, ml.NetNsPath) (type *macvtapDevicePlugin) as type dpm.PluginInterface in return argument:
	*macvtapDevicePlugin does not implement dpm.PluginInterface (missing GetPreferredAllocation method)
```
which is why I'm currently sticking with an older `k8s.io/kubelet v0.18.8`

**Release Note**
```release-note
Bump github.com/aktau/github-release to v0.8.1
Bump github.com/containernetworking/cni to v0.8.0
Bump kubernetes packages to v1.19.1
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,49 +3,54 @@ module github.com/kubevirt/macvtap-cni
...
-	k8s.io/apimachinery v0.0.0
-	k8s.io/client-go v0.0.0
-	k8s.io/kubernetes v0.0.0-00010101000000-000000000000
+	golang.org/x/net v0.0.0-20200822124328-c89045814202
+	golang.org/x/tools v0.0.0-20200616133436-c1934b75d054
+	google.golang.org/grpc v1.31.0
+	k8s.io/api v0.19.1
+	k8s.io/apimachinery v0.19.1
+	k8s.io/client-go v0.19.1
+	k8s.io/kubelet v0.18.8
```
Comment (phoracek) [inline go.mod:29]: Why is kubelet older than kubernetes here? And why is it required?

Comment (rhrazdil) [reply go.mod:29]: deviceplugin has moved under kubelet
With v0.19.1 there is an error:
```
pkg/deviceplugin/lister.go:168:31: cannot use NewMacvtapDevicePlugin(c.Name, c.Master, c.Mode, c.Capacity, ml.NetNsPath) (type *macvtapDevicePlugin) as type dpm.PluginInterface in return argument:
	*macvtapDevicePlugin does not implement dpm.PluginInterface (missing GetPreferredAllocation method)

```
Probably related to device scheduler in 1.19.

Comment (phoracek) [reply go.mod:29]: Puff, we may need to bump that on device-plugin-manager repo... But that can wait for a follow-up, let's not get even deeper into this. Ack, thanks

**Review by phoracek — APPROVED** [review]
/lgtm

Let's hear from @maiqueb too

**Comment by phoracek** [comment]
Or not, where did that approve came from -_-

**Comment by maiqueb** [comment]
Yeah, something is awfully wrong here.

From the bot's words, I can see that the PR has been auto-approved by whoever proposed it - https://github.com/kubevirt/macvtap-cni/pull/29#issuecomment-694134880

That wrong on so many levels ... 

@phoracek please remember that until we fix this mess, you gotta `hold` if you want more inputs.

**Comment by phoracek** [comment]
Hold my hand :duck: /\ :duck:

**Pull request merged**